pyrobosim/pyrobosim/utils/polygon.py

```diff
@@ -205,7 +204,7 @@ def polygon_and_height_from_mesh(mesh_data):
     return (Polygon(hull_pts), height)
 
 
-def sample_from_polygon(polygon, max_tries=100):
+def sample_from_polygon(polygon, max_tries=100)->None:
```

Comment:
This nominally returns a tuple, so it should be e.g. `typing.Tuple[float, float]` if anything ... or really `typing.Tuple[typing.Optional(float), typing.Optional(float)]`